src/Adapter/PlatformServices.Universal/PlatformServices.Universal.csproj

```diff
@@ -48,13 +35,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="$(MicrosoftNETTestSdkVersion)" />
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="$(MicrosoftNETCoreUniversalWindowsPlatformVersion)" />
-    <!-- CVE-2017-0247 -->
-    <PackageReference Include="System.Net.WebSockets.Client" Version="$(SystemNetWebSocketsClientVersion)" />
```

Comment:
Some of these dependencies are also declared in nuget package. Shall we remove them too?